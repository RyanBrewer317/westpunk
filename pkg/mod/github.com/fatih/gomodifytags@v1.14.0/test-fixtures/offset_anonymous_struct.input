package main

func test(arg struct {
	Field int
}) {
}
