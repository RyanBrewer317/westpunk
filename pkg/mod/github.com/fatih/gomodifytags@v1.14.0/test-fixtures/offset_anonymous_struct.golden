package main

func test(arg struct {
	Field int `json:"field"`
}) {
}
